traveler_app/src/main/java/com/guestlogix/traveler/fragments/SettingsFragment.java

```diff
@@ -70,6 +70,7 @@ public void onDisplayPreferenceDialog(Preference preference) {
 
             EditUrlPreference editUrlPreference = (EditUrlPreference) preference;
             // check if dialog is already showing
+            //TODO: change deprecated api
```

Comment:
make your TODO comments more informative. "Change deprecated API" does not provide enough context to new developers coming in.